docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
+## Overview
+
+Fees are pooled separately and withdrawn lazily, at any time.  They are not
+bonded, and can be paid in multiple tokens.  An adjustment factor is maintained
+for each validator and delegator to determine the true proportion of fees in
+the pool they are entitled too.  Adjustment factors are updated every time a
+validator or delegator's voting power changes.  Validators and delegators must
+withdraw all fees they are entitled too before they can bond or unbond Atoms.
+
+## Affect on Staking
+
+Because fees are optimized to note
+
+Commission on Atom Provisions and having atoms autobonded are mutually
+exclusive (we can’t have both). The reason for this is that if there are atoms
+commissions and autobonding, the portion of atoms the fee distribution
+calculation would become very large as the atom portion for each delegator
+would change each block making a withdrawal of fees for a delegator require a
+calculation for every single block since the last withdrawal. Conclusion we can
+only have atom commission and unbonded atoms provisions, or bonded atom
+provisions and no atom commission
+
+## Fee Calculations
+
+Collected fees are pooled globally and divided out passively to validators and
+delegators. Each validator has the opportunity to charge commission to the
+delegators on the fees collected on behalf of the delegators by the validators.
+Fees are paid directly into a global fee pool. Due to the nature of of passive
+accounting whenever changes to parameters which affect the rate of fee
```

Comment:
"*accounting,* whenever changes to parameters which affect the rate of fee distribution *occur*, ..."

Reply:
WIP, next PR